### PR TITLE
Add refresh button for current mailbox

### PR DESCRIPTION
**Pull request by GretaD**
Fixes https://github.com/nextcloud/mail/issues/4456

### CODE REVIEW

File: src/components/Navigation.vue [hunk 1]
```diff
@@ -27,6 +27,9 @@
 			button-class="icon-add"
 			role="complementary"
 			@click="onNewMessage" />
+		<button v-if="currentMailbox"
```
Comment (ChristophWurst) [inline src/components/Navigation.vue:30]: can we also visually disable the button during a sync?

File: src/components/Navigation.vue [hunk 2]
```diff
@@ -166,9 +180,46 @@ export default {
...
+				this.refreshing = false
+			}
+		},
 	},
 }
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
+::v-deep .button {
```
Comment (ChristophWurst) [inline src/components/Navigation.vue:203]: are you sure v-deep is required here?

**Review by ChristophWurst — APPROVED** [review]
Code looks good!

File: src/components/Navigation.vue (hunk 2 above)
Comment (GretaD) [reply src/components/Navigation.vue]: of course not

**Comment by StCyr** [comment]
Could you add a little animation?

cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/mail$ git diff src/
diff --git a/src/components/Navigation.vue b/src/components/Navigation.vue
index 4a75baa39..b41eda99a 100644
--- a/src/components/Navigation.vue
+++ b/src/components/Navigation.vue
@@ -218,9 +218,20 @@ export default {
        &:disabled {
                cursor: not-allowed;
                opacity: .5;
+               animation: rotation infinite 2s linear;
        }
 }
 ::v-deep .app-navigation-new button {
        width: 250px !important;
 }
+
+@keyframes rotation{
+  from{
+    transform:rotate(-0deg);
+  }
+
+  to{
+    transform:rotate(-360deg);
+  }
+}
 </style>

**Review by StCyr — APPROVED** [review]
Let's merge!

**Pull request merged**